code/game/objects/items/tools/soldering_tool.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/tool/solderingtool
+	name = "soldering tool"
+	desc = "A hand tool to fix combat robot's trauma. You do not need welding goggles for this and you need medical skills for this."
+	icon = 'icons/obj/items/surgery_tools.dmi'
+	icon_state = "alien_hemostat"
+	w_class = WEIGHT_CLASS_SMALL
+	var/skill_level_needed = SKILL_MEDICAL_PRACTICED
+	var/unskilled_delay = SKILL_TASK_EASY
+
+/obj/item/tool/solderingtool/attack(mob/living/carbon/human/H, mob/user)
```

Comment:
I know its copy-pasta, but can you not have alphabet soup param names.

Reply:
Not too sure how to fix this.